engine/core/src/main/java/org/pentaho/reporting/engine/classic/core/layout/text/TextUtility.java

```diff
@@ -62,4 +62,27 @@ public static ExtendedBaselineInfo createBaselineInfo( final int codepoint, fina
         fontMetrics.getMaxHeight(), underlinePosition, strikeThroughPosition );
   }
 
+  public static ExtendedBaselineInfo createCorrectBaselineInfo( final int codepoint, final FontMetrics fontMetrics,
```

Comment:
I'd name this function 'createPaddedBaselineInfo' instead. If you say create correct bla, then I'd wonder is the the other method not correct? 

Also: This method is unlogical for the causal reader (head voice: "why the heck would they do that?!") so it might be worthwhile to add a extra comment here:

"This method is a workaround for broken font metrics supplied by some truetype fonts. Those fonts trip up Excel's row heights in the workbooks. We therefore try to detect those fonts by checking whether their declared sizes are awfully close to the requested font size and apply some extra padding in that case. For all details please read PRD-5435."

This just makes sure that the next person touching that code gets an idea of the madness that goes on without having to dig into a long conversation chain on the case. This then also gives context to the magic numbers you have in your code (and thus frees you from having them as named constants, which for a single use like that always feels over the top.